Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN cd /tmp && \
 
 # Get nginx-rtmp module.
 RUN cd /tmp && \
-  wget https://github.com/arut/nginx-rtmp-module/archive/v${NGINX_RTMP_VERSION}.tar.gz && \
+  wget https://github.com/chetandhembre/nginx-rtmp-module/archive/v${NGINX_RTMP_VERSION}.tar.gz && \
```

Comment:
I like the idea of using a forked project that is actively maintained. However, I want to keep the `latest` tag on this project true to the original `nginx-rtmp-module` source. Can we instead create a new `Dockerfile.chetandhembre` file where I can build and push this as a separate tag?